pandas/tests/groupby/aggregate/test_numba.py

```diff
@@ -78,13 +80,14 @@ def func_numba(values, index):
     tm.assert_equal(result, expected)
 
 
-@td.skip_if_no("numba")
 @pytest.mark.filterwarnings("ignore")
 # Filter warnings when parallel=True and the function can't be parallelized by Numba
 @pytest.mark.parametrize("jit", [True, False])
 @pytest.mark.parametrize("pandas_obj", ["Series", "DataFrame"])
 def test_cache(jit, pandas_obj, nogil, parallel, nopython):
     # Test that the functions are cached correctly if we switch functions
+    pytest.importorskip("numba")
```

Comment:
does this change the collection/running performance of the test suite?

Reply:
Collection locally I'm getting 35.19s on main vs 34.13s on the PR so negligible. Based on the job runtimes doesn't seem noticeably different